modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -3,54 +3,55 @@ ifndef::env-site,env-github[]
+The Try scope can distinguish among various error type conditions and apply different behaviors. If an error is raised by a component inside a Try scope, then the Try scope's error handler is executed. At this point, the error is available for inspection, so the handlers can execute and act accordingly:
 
-* An On Error Continue executes and sends the result of the execution to its container Try scope, which uses that result to complete the execution successfully. Any transactions at this point are also committed.
-* An On Error Propagate rolls back any transactions, then executes and uses that result to re-throw the existing error, meaning that its container Try Scope's execution fails.
+* On Error Continue
++
+Executes and sends the result of the execution to its container Try scope, which uses that result to complete the execution successfully. Any transactions at this point are also committed.
+* On Error Propagate
++
+Rolls back any transactions, then executes and uses that result to re-throw the existing error, causing its container Try scope's execution to fail.
 
-If the Try scope has several components, then once a component raises an exception, subsequent components in the Try scope are skipped, regardless of the type of error handler that catches the exception. In the case of On Error Propagate, the error is propagated to the flow's error handler, as if the Try scope did not exist. In the case of On Error Continue, processing continues outside the Try scope at the next downstream flow component, as if the Try scope never threw an exception.
+Example Try Scope Configuration
 
-In the previous example, any database connection errors (`DB:CONNECTIVITY`) are propagated because of the On Error Propagate (`on-error-propagate`) setting. Propagation of this error causes the Try process to fail and the flow’s error handler to execute. Other errors are handled through On Error Continue (`on-error-continue`), so the Try is treated as successful when they occur, meaning that the next processor, an HTTP request, continues its execution.
+image::error-handling-try-scope.png[Error Handling with the Try Scope]
 
-Within your flow, you can group a set of event processors that might throw an error inside a Try Scope. The Try Scope allows you to enclose various event processors in your flow and assign it an Error Handler scope in case that operation fails. +
+In the previous example, any database connection errors (`DB:CONNECTIVITY`) are propagated because of the On Error Propagate (`on-error-propagate`) error handler. Propagation of this error causes the Try process to fail and the flow’s error handler to execute. Other errors are handled through On Error Continue (`on-error-continue`), so the Try is treated as successful when they occur, meaning that the next operation, an HTTP request, continues its execution.
```

Comment:
causes the Try process --> cause the Try scope process
so the Try is treated as successful --> so the Try scope is treated